deploy/csi-s3/values.yaml

```diff
@@ -1,13 +1,32 @@
 ---
 images:
+  defaultImage:
+    registry: infracreate-registry.cn-zhangjiakou.cr.aliyuncs.com
+    pullPolicy: IfNotPresent
   # Source: quay.io/k8scsi/csi-attacher:v3.0.1
-  attacher: infracreate-registry.cn-zhangjiakou.cr.aliyuncs.com/apecloud/csi-attacher:v3.4.0
+  attacher:
+    registry: ""
+    repository: apecloud/csi-attacher
+    tag: v3.4.0
+    pullPolicy: ""
   # Source: quay.io/k8scsi/csi-node-driver-registrar:v1.2.0
-  registrar: infracreate-registry.cn-zhangjiakou.cr.aliyuncs.com/apecloud/csi-node-driver-registrar:v2.5.1
+  registrar:
+    registry: ""
+    repository: csi-node-driver-registrar
```

Comment:
apecloud is missing

Reply:
fixed
![image](https://github.com/apecloud/kubeblocks/assets/7733385/ab4349af-5023-4e51-b7de-b7ba1cea55ed)